client/index.jade

```diff
@@ -0,0 +1,15 @@
+doctype html
+html
+
+  head
+    title Hello, React
+
+  body
+
+    h1.alert.alert-info.this-works webpack dev server (with HMR)
+    #app
+
+    script(src="vendor-bundle.js")
+    script(src="app-bundle.js")
+    script.
+      ReactDOM.render(React.createElement(HelloWorldApp, {name: "Stranger"}), document.getElementById('app'))
```

Comment:
I don't understand how this works, and what I'm supposed to change here. How can a static mock possibly replicate what the real app can be?

For example, my app uses react-router to determine _what_ component to render from the URL— which this mock can't replicate.